miio/integrations/roborock/vacuum/vacuumcontainers.py

```diff
@@ -252,7 +253,7 @@ def dock_error(self) -> Optional[str]:
             return "Definition missing for dock error %s" % self.dock_error_code
 
     @property
-    @sensor("Battery", unit="%", device_class="battery", enabled_default=False)
+    @sensor("Battery", unit="%", id=VacuumId.Battery, device_class="battery", enabled_default=False)
```

Comment:
```suggestion
    @sensor("Battery", unit="%", id=VacuumId.Battery)
```
No need to define these anymore as we can deduce that directly from the id.

Otherwise this looks good to go as soon as the CI passes, thanks!